libs/blocks/media/media.js

```diff
@@ -49,6 +49,7 @@ export default function init(el) {
     if (image) image.classList.add('image');
     const img = image.querySelector(':scope img');
     if (header && img?.alt === '') img.alt = header.textContent;
+    if (image.querySelector('video')) applyHoverPlay(image.querySelector('video'));
```

Comment:
nit: only run selector once
```suggestion
    const imageVideo = image.querySelector('video');
    if (imageVideo) applyHoverPlay(imageVideo);
```

Reply:
Thanks for the review @chrischrischris . This is done.